src/edu/csus/ecs/pc2/core/ParseArguments.java

```diff
@@ -11,6 +11,8 @@
 import java.util.Set;
 import java.util.Vector;
 
+import edu.csus.ecs.pc2.AppConstants;
```

Comment:
It doesn't appear that this import is used/needed.

Reply:
agreed, remove unused import (this was not found on develop branch)

Reply:
Will be removed in latest push.